x-pack/plugins/reporting/server/routes/common/get_user.ts

```diff
@@ -5,9 +5,9 @@
  * 2.0.
  */
 
+import { SecurityRequestHandlerContext } from '@kbn/core-security-server';
 import { KibanaRequest } from '@kbn/core/server';
-import { SecurityPluginStart } from '@kbn/security-plugin/server';
 
-export function getUser(request: KibanaRequest, security?: SecurityPluginStart) {
-  return security?.authc.getCurrentUser(request) ?? false;
+export function getUser(request: KibanaRequest, security?: SecurityRequestHandlerContext) {
+  return security?.authc.getCurrentUser() ?? false;
```

Comment:
I think these can be made to be non-optional
```suggestion
export function getUser(request: KibanaRequest, security: SecurityRequestHandlerContext) {
  return security.authc.getCurrentUser() ?? false;
```